public/configurations/queries/aar-enterprise-nsg-app-cumulative.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id": "aar-enterprise-nsg-app-cumulative",
+    "title": "Data Usage per NSG for an App",
+    "service": "elasticsearch",
+    "query": {
+        "index": "{{index:nuage_dpi_flowstats}}",
+        "type": "{{type:nuage_doc_type}}",
+        "body": {
+            "size": 0,
+            "query": {
+                "bool": {
+                    "must": [
+                        {
+                            "range": {
+                                "timestamp": {
+                                    "gte": "{{startTime:now-24h}}",
+                                    "lte": "{{endTime:now}}",
+                                    "format": "epoch_millis"
+                                }
+                            }
+                        },
+                        {
+                            "term": {
+                                "IngressBytes": 0
+                            }
+                        },
+                        {
+                            "term": {
+                                "EnterpriseName": "{{enterpriseName:test_org}}"
+                            }
+                        },
+                        {
+                            "term": {
+                                "L7ClassEnhanced":"{{appName}}"
+                            }
+                        }
+                    ]
+                }
+            },
+             "aggs": {
+                "SourceNSG": {
```

Comment:
This should actually be a term query coming from nsg-per-app graph. And you need to aggregate users instead